### PR TITLE
[O2B-1346] API for runs filtering by INEL and mu values

### PR DESCRIPTION
**Pull request by xsalonx**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected

Notable changes for users:
- NA

Notable changes for developers:
- Added filtering by INEL and mu values to runs API

Changes made to the database:
- NA

### CODE REVIEW

File: lib/domain/dtos/filters/NumericalComparisonDto.js [hunk 1]
```diff
@@ -13,7 +13,11 @@
 
 const Joi = require('joi');
 
-exports.NumericalComparisonDto = Joi.object({
-    operator: Joi.string().valid('<', '<=', '=', '>=', '>'),
+const SYMBOLIC_NUMERICAL_COMPARISON_OPERATORS = ['<', '<=', '=', '>=', '>'];
```
Comment (martinboulais) [inline lib/domain/dtos/filters/NumericalComparisonDto.js:16]: I would remove the `SYMBOLIC_`

File: lib/usecases/run/GetAllRunsUseCase.js [hunk 2]
```diff
@@ -164,6 +170,40 @@ class GetAllRunsUseCase {
...
+                 * The computed mu of inelastic interaction rate
+                 *
+                 * @param {function} literal function to create an object representing a database literal expression
+                 * @return {Object} the object representing the column
+                 */
+                const computedColumn = ({ literal }) =>
+                    literal(`\`Run\`.\`inelastic_interaction_rate_avg\` / (
+                        SUBSTRING_INDEX(SUBSTRING_INDEX(\`lhcFill\`.\`filling_scheme_name\`, '_', 4), '_', -1) 
+                        * ${PhysicalConstant.LHC_REVOLUTION_FREQUENCY_HZ})
+                    `);
```
Comment (martinboulais) [inline lib/usecases/run/GetAllRunsUseCase.js:185]: it might be worth using a SQL virtual column?

**Review by martinboulais — CHANGES_REQUESTED** [review]

File: lib/usecases/run/GetAllRunsUseCase.js (hunk 2 above)
Comment (xsalonx) [reply lib/usecases/run/GetAllRunsUseCase.js:185]: There is reference to another table,  a view would be needed

File: lib/domain/dtos/filters/NumericalComparisonDto.js (hunk 1 above)
Comment (xsalonx) [reply lib/domain/dtos/filters/NumericalComparisonDto.js]: applied

File: lib/usecases/run/GetAllRunsUseCase.js (hunk 2 above)
Comment (martinboulais) [reply lib/usecases/run/GetAllRunsUseCase.js:185]: Ah yes indeed I read too quickly

**Review by martinboulais — APPROVED** [review]

**Pull request merged**